### DISCUSSION TITLE
Integrazione antifurto Daitem e  Home Assistant

### DISCUSSION THREAD

**Discussion by alessandrobenevelli** [indomus/forum#1939] [GENERALE - Discussioni generiche]
Buonasera a tutti,
qualcuno ha esperienza riguardo l'integrazione di una centrale d'allarme Daitem e-Nova e Home Assistant? E' possibile un collegamento, anche passando dal cloud Daitem ed ottenere così le info riguardo lo stato dell'allarme e dei singoli sensori (apertura varchi, movimento tapparelle, sensore di movimento, ecc...)?

Grazie a tutti
Alessandro

**Comment by sbellisario** [comment]
Da possessore di un antifurto Daitem, mi chiedevo la stessa cosa...
Sarebbe molto interessante, anche in virtù del fatto che l'app Daitem è un ciofeca assurda.

Ciao

**Comment by pippoduk** [comment]
ragazzi io invece ho un antifurto domotico AXEL pero' il myweb è una ciofeca, si sa se si potrebbe integrare con IFTT o con HA?
